cypress/scripts/run-starter-prototype.js

```diff
@@ -11,7 +11,8 @@ const testDir = path.resolve(process.env.KIT_TEST_DIR || defaultKitPath)
   await mkPrototype(testDir, { overwrite: true, allowTracking: false })
   await installExtensions(testDir, [
```

Comment:
I think this is good enough for now but quite quickly we should look at installing and uninstalling extensions during test runs.  I'm going to create a ticket for being able to add/remove extensions while the kit is running.